libvast/src/arrow_table_slice.cpp

```diff
@@ -6,9 +6,8 @@
 // SPDX-FileCopyrightText: (c) 2019 The VAST Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
-#include "vast/arrow_table_slice.hpp"
-
-#include "vast/arrow_table_slice_builder.hpp"
+#include "vast/arrow_ng_table_slice.hpp"
+#include "vast/arrow_ng_table_slice_builder.hpp"
 #include "vast/config.hpp"
```

Comment:
I'm confused; I would have expected the new `arrow_ng_table_slice` to be only imported in the new file(s), not for the existing arrow table slice.

Reply:
The files were just swapped—I renamed in the wrong copy. I already fixed this in a force-push.